### PR TITLE
new template marketing channels

### PR DESCRIPTION
**Pull request by MonikaFeigler**

### CODE REVIEW

**Review by MonikaFeigler — COMMENTED** [review]
I don't know which icon to use in repository since the whole purpose of this template is to use at least one marketing channels and it can be one of:
facebook ads, google ads, linkedin ads or bing ads (but this part is under testing)

**Comment by webrouse** [comment]
`I don't know which icon to use in repository since the whole purpose of this template is to use at least one marketing channels and it can be one of:` 

Change is planned and it will be possible to define more icons at the template level, the UI will draw an arrow between them. 

It will probably be part of the next sprint.

File: .keboola/repository.json [hunk 1]
```diff
@@ -154,6 +154,21 @@
           "path": "v0"
         }
       ]
+    },
+    {
+      "id": "keboola-marketingchannels",
+      "icon": "common:settings",
+      "name": "KEBOOLA-MARKETINGCHANNELS",
+      "description": "Full workflow to ...",
```
Comment (webrouse) [inline .keboola/repository.json:162]: Please fill in description.

Comment (MonikaFeigler) [reply .keboola/repository.json]: done, sorry, forgot, thanks for the check :-)

**Review by webrouse — APPROVED** [review]
OK. You can merge it, the API / UI will be updated every 5 minutes, after merge.

Please post to the news feed channel (always).

**Review by webrouse — APPROVED** [review]

**Pull request merged**